packages/mermaid/src/docs/syntax/classDiagram.md

```diff
@@ -560,6 +560,38 @@ classDiagram
   style Mineral fill:#bbf,stroke:#f66,stroke-width:2px,color:#fff,stroke-dasharray: 5 5
 ```
 
+### Font Customization with `fontFamily`
+
+Mermaid allows you to customize the font used in your class diagrams using the `fontFamily` property under `themeVariables`. This applies the specified font across the entire diagram, affecting node labels and annotations.
+
+You can use any valid CSS `font-family` string. Be sure the font is available in the viewer’s environment (e.g., a standard web-safe font or one loaded via a stylesheet).
+
+#### Example
+
+```mermaid
+%%{ init: { "theme": "default", "themeVariables": { "fontFamily": "Comic Sans MS" } } }%%
```

Comment:
We've deprecated using init in the examples, please use the frontmatter config. 

https://github.com/mermaid-js/mermaid/pull/6499